ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh

```diff
@@ -23,6 +23,22 @@ third_octet=$(grep -oP '[ci|qe\-discon]-segment-\K[[:digit:]]+' <(echo "${LEASED
 
 export HOME=/tmp
 
+if [[ ${LEASED_RESOURCE} == *"vlan"* ]]; then
+  vlanid=$(grep -oP '[ci|qe\-discon]-vlan-\K[[:digit:]]+' <(echo "${LEASED_RESOURCE}"))
+  dns_server=$(jq -r --arg VLANID "$vlanid" '.[$VLANID].dnsServer' /var/run/vault/vsphere-config/subnets.json)
+  gateway=$(jq -r --arg VLANID "$vlanid" '.[$VLANID].gateway' /var/run/vault/vsphere-config/subnets.json)
+  cidr=$(jq -r --arg VLANID "$vlanid" '.[$VLANID].cidr' /var/run/vault/vsphere-config/subnets.json)
+
+  # ** NOTE: The first two addresses are not for use. [0] is the network, [1] is the gateway
+  rendezvous_ip_address=$(jq -r --arg VLANID "$vlanid" '.[$VLANID].ipAddresses[4]' /var/run/vault/vsphere-config/subnets.json)
```

Comment:
should we check for missing VLANs in the Vault config? 
replace /var/run/vault/vsphere-config/subnets.json with a  var?